yaas/__main__.py

```diff
@@ -60,9 +62,9 @@ def main():
     args, extra = parser.parse_known_args(sys.argv[1:])
     config.args = args
 
-    if args.command != 'version' and config.server_url is None:
-        parser.error("Ambari server URL must be specified " \
-            "through environment variable YAAS_SERVER_URL")
+    if args.command != 'version' and config.server is None:
```

Comment:
We discussed this in the design phase, but I again think the default for server should be localhost. That's the default for ambari agents, unless you edit their config files and point them to an external server.